src/client/state/RoomsInput.js

```diff
@@ -205,8 +205,8 @@ class RoomsInput extends EventEmitter {
   //
   // This includes inserting any custom emoji that might be relevant, and (only if the
   // user has enabled it in their settings) formatting the message using markdown.
-  formatAndEmojifyText(text) {
-    const allEmoji = getShortcodeToEmoji(this.matrixClient);
+  static formatAndEmojifyText(room, text) {
```

Comment:
`sendEditedMessage`  calls it with `this.`formatAndEmojifyText(...) at line 407. 

If it doesn't access `this` anymore we can place it outside class.